mass2/core/channel.py

```diff
@@ -493,7 +493,7 @@ def rough_cal(  # noqa: PLR0917
         line_names: list[str | float],
         uncalibrated_col: str = "filtValue",
         calibrated_col: str | None = None,
-        use_expr: pl.Expr = field(default_factory=alwaysTrue),
+        use_expr: pl.Expr = pl.lit(True),
```

Comment:
How sure are you that this is good? I remember there's a reason we used the `default_factory` feature.... Aha! See #35 in this repo. We discussed on August 19 that all instances sharing the same object `pl.lit(True)` confuses the type checker, because it cannot tell that the expression is immutable.

On the other hand, that discussion makes me think you should be failing tests or `mypy`. And apparently you aren't. ???